### ISSUE TITLE
Best practice for users to get the binary installed?

### ISSUE THREAD

**Issue by sckott** [sckott/rt-cli#7]
@JosiahParry if a rust package is not on crates.io, what's best practice for having users install a rust cli tool like this? Is it maybe `cargo install --git https://github.com/sckott/rt-cli.git` ? Or something else? Build binaries and add them to releases in the repo?

**Comment by JosiahParry** [comment]
Yes that is right! There is also a library called cargo dist (which I haven’t used yet) that helps create installers for various OS

https://github.com/axodotdev/cargo-dist

**Comment by sckott** [comment]
Thanks
